neqo-transport/src/send_stream.rs

```diff
@@ -741,7 +744,7 @@ impl SendStream {
             sendorder: None,
             bytes_sent: 0,
             fair: false,
-            writable_event_low_watermark: 1.try_into().unwrap(),
+            writable_event_low_watermark: NonZeroUsize::new(1).expect("1 != 0"),
```

Comment:
```suggestion
            writable_event_low_watermark: NonZeroUsize::MIN,
```

Feel free to ignore.

https://doc.rust-lang.org/std/num/struct.NonZero.html#associatedconstant.MIN